lib/iris/experimental/ugrid.py

```diff
@@ -615,6 +615,437 @@ def equal(self, other, lenient=None):
+#             node_dimension=None,
+#             edge_dimension=None,
+#             face_dimension=None,
+#             node_coords_and_axes=None,   # [(coord, "x"), (coord, "y")] this is a stronger contract, not relying on guessing
+#             edge_coords_and_axes=None,   # ditto
+#             face_coords_and_axes=None,   # ditto
+#             connectivities=None,         # [Connectivity, [Connectivity], ...]
+#     ):
+#         # TODO: support volumes.
+#         # TODO: support (coord, "z")
+#
+#         # These are strings, if None is provided then assign the default string.
+#         self.node_dimension = node_dimension
+#         self.edge_dimension = edge_dimension
+#         self.face_dimension = face_dimension
+#
+#         self._metadata_manager = metadata_manager_factory(MeshMetadata)
+#
+#         self._metadata_manager.topology_dimension = topology_dimension
+#
+#         self.standard_name = standard_name
+#         self.long_name = long_name
+#         self.var_name = var_name
+#         self.units = units
+#         self.attributes = attributes
+#
+#         # based on the topology_dimension create the appropriate coordinate manager
+#         # with some intelligence
+#         self._coord_manager = ...
+#
+#         # based on the topology_dimension create the appropriate connectivity manager
+#         # with some intelligence
+#         self._connectivity_manager = ...
+#
+#     @property
+#     def all_coords(self):
+#         # coords = mesh.all_coords
+#         # coords.face_x, coords.edge_y
+#         pass
+#
+#     @property
+#     def node_coords(self):
+#         # perhaps return a namedtuple?
+#         # this would give:
+#         # node_coords = mesh.node_coords
+#         # node_coords.x
+#         # node_coords.y
+#         pass
+#
+#     @property
+#     def edge_coords(self):
+#         # as above
+#         pass
+#
+#     @property
+#     def face_coords(self):
+#         # as above
+#         pass
+#
+#     @property
+#     def all_connectivities(self):
+#         # conns = mesh.all_connectivities
+#         # conns.edge_node, conns.boundary_node
+#         pass
+#
+#     @property
+#     def face_node_connectivity(self):
+#         # required
+#         return self._connectivity_manager.face_node
+#
+#     @property
+#     def edge_node_connectivity(self):
+#         # optionally required
+#         return self._connectivity_manager.edge_node
+#
+#     @property
+#     def face_edge_connectivity(self):
+#         # optional
+#         return self._connectivity_manager.face_edge
+#
+#     @property
+#     def face_face_connectivity(self):
+#         # optional
+#         return self._connectivity_manager.face_face
+#
+#     @property
+#     def edge_face_connectivity(self):
+#         # optional
+#         return self._connectivity_manager.edge_face
+#
+#     @property
+#     def boundary_node_connectivity(self):
+#         # optional
+#         return self._connectivity_manager.boundard_node
```

Comment:
Typo?  I think this should be:
return self. ...   .boundary_node  # not boundard_node

Reply:
There's nothing we can do here since the PR is already merged.
@bjlittle has made a number of corrections in #4005, including the typos. @abooton if you spot any more, then #4005 is the place to comment.